src/app/rundown/components/segment-end-indicator/segment-end-indicator.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, HostBinding, Input } from '@angular/core'
+import { Part } from '../../../core/models/part'
+
+@Component({
+  selector: 'sofie-segment-end-indicator',
+  templateUrl: './segment-end-indicator.component.html',
+  styleUrls: ['./segment-end-indicator.component.scss'],
+  host: {
+    class: 'c-segment-end-indicator',
+  },
+})
+export class SegmentEndIndicatorComponent {
+  @Input()
+  public lastPartInSegment?: Part
+
+  @HostBinding('class.c-segment-end-indicator--active')
+  public get isLastPart() { return this.lastPartInSegment?.isOnAir ?? false }
```

Comment:
From the name of the getter, with the knowledge that i have supplied a part to an input called `lastPartInSegment`, i'd guess that this would always return true.
Even if the last part is not on air, it is still the last part.

Also adding return type to the getter, if possible, would be nice.

Reply:
It will only return true when the last Part is on Air.
A better name for the getter would be `isLastPartOnAir`

Reply:
Adding return types gave some issues with some versions of these getters, but after trying it again just now it didn't complain. I don't remember if this was an issue here @KvelaGorrrrnio?

Reply:
That was bad naming. I have changed it to `isLastPartOnAir` and added return type.
We had funny issues where it wouldn't compile if we provided a return type – but it works here.